packages/renderer/src/navigation.ts

```
@@ -169,5 +169,8 @@ export const handleNavigation = (request: InferredNavigationRequest<NavigationPa
     case NavigationPage.SECRETS:
       router.goto(`/secrets`);
       break;
+    case NavigationPage.SECRET_CREATE:
+      router.goto(`/secrets/create`);
```

Biome (2.5.1)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)